src/domain/ruleset-repository.ts

```diff
@@ -170,15 +197,25 @@ export class RulesetRepository extends Repository {
   public get config(): Configuration {
     return this._config;
   }
+
+  public getModuleName(moduleRoot: string): string {
+    return new ModuleFile(path.join(this.diskPath, moduleRoot, "MODULE.bazel"))
+      .moduleName;
+  }
 }
 
-function validateMetadataTemplate(rulesetRepo: RulesetRepository) {
+function validateMetadataTemplate(
+  rulesetRepo: RulesetRepository,
+  moduleRoot: string
+) {
   let metadata: Record<string, unknown>;
+
   try {
     metadata = JSON.parse(
-      fs.readFileSync(rulesetRepo.metadataTemplatePath, "utf-8")
+      fs.readFileSync(rulesetRepo.metadataTemplatePath(moduleRoot), "utf-8")
     );
   } catch (error) {
+    // TODO: module root in error message
```

Comment:
Did you want to address this before merging?

Reply:
Good catch. Fixed.